### DISCUSSION TITLE
Need help to get HC ROMs to appear in the Downloads section of Stella

### DISCUSSION THREAD

**Discussion by supremeai** [stella-emu/stella#1000] [Q&A] [answered]
On the Mac, the 1st time we clicked the Downloads folder, Mac asked for permission to access the Downloads, we accidentally click "don't allow" and now we can't get Mac to get access the Downloads where the HC ROMs are. We need to have the HC ROMs show up in Stella and are grateful to know how to accomplish this.

We uninstalled Stella and re-insalled Stella, and the Mac seems to remember that we clicked "don't allow" for the Downloads access permissions.

**Comment by thrust26** [comment]
I am very sure, that his is solely a MacOS problem and Stella cannot do anything here. I suggest that you ask for help in a Mac forum.

**Comment by DirtyHairy** [comment, marked as answer]
Just go to system settings -> security, select the relevant permission class (I think it'll be "files and folders") and check the slider for Stella. But as Thomas @thrust26 says, this is not specific to Stella, but rather how permissions on MacOS work for every app.

**Comment by supremeai** [comment]
Thanks! Got it working. 

Question: My Xbox controller is bluetooth connected to the Mac, and is there a way to use it rather than the keyboard?
